src/components/MultiSelect/MultiSelect.stories.mdx

```diff
@@ -0,0 +1,297 @@
+import { getCategory } from "../../utils/componentCategories";
+import { useState } from "react";
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import MultiSelect from "./MultiSelect";
+import useMultiSelect from "../../hooks/useMultiSelect";
+import * as stories from "./MultiSelect.stories.tsx";
+
+<Meta
+  title={getCategory("MultiSelect")}
+  component={MultiSelect}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=43593%3A24611",
+    },
+    jest: ["MultiSelect.test.tsx"],
+  }}
+  argTypes={{
+    label: { table: { default: "" } },
+    variant: { table: { default: "listbox" }, control: false },
+    items: { control: false },
+    selectedItems: { control: false },
+    width: { table: { default: "default" } },
+    defaultIsOpen: { table: { default: false } },
+    isBlockElement: { table: { default: false } },
+    onApply: {
+      description:
+        "The action to perform for save/apply button of multiselect. `onApply: () => void;`",
+    },
+    onChange: {
+      description:
+        "The action to perform on the checkbox's onChange function. <br /> For dialog: `(event: React.ChangeEvent<HTMLInputElement>) => void` <br /> For listbox: `onChange: (selectedItem: MultiSelectItem, id: string) => void;`",
+    },
+    onMixedStateChange: {
+      description:
+        "The action to perform for a mixed state checkbox (parent checkbox). `onMixedStateChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;`",
+    },
+  }}
+/>
+
+# MultiSelect
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.1.0`    |
+| Latest            | `1.1.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Listbox Variant](#multiselect-listbox-variant)
+- [Dialog Variant](#multiselect-dialog-variant)
```

Comment:
The link works now but this should match the title of the heading so it should be "MultiSelect Dialog Variant".